generics/middleware/authenticator.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * name : middleware/authenticator.js
+ * author : Aman Karki
+ * Date : 20-July-2020
+ * Description : Keycloak authentication.
+ */
+
+// dependencies
+const jwtDecode = require('jwt-decode');
```

Comment:
@rakeshSgr - Please clean this file to include only verification code. Also provision for which paths to work with token or internal access token